src/OfficeToolPlus/Style/Languages/de-de.xaml

```diff
@@ -338,11 +338,11 @@
     <sys:String x:Key="ToolboxRepairVBS">VBS-Dateizuordnungen reparieren</sys:String>
     <sys:String x:Key="ToolboxRepairVBSDescription">Hiermit wird die VBS-Dateizuordnung wiederhergestellt. Dies betrifft die Fehlermeldung "Es gibt keine Skript-Engine für die Dateierweiterung ".vbs"" bei der Aktivierung von Office.</sys:String>
     <sys:String x:Key="ToolboxRepairVariables">System-Variablen reparieren</sys:String>
-    <sys:String x:Key="ToolboxRepairVariablesDescription">This will reset your system variables to default values. Do not use it if you are unsure.&#x000A;&#x000A;After the repair is complete, you need to reboot your system for it to take effect.</sys:String>
-    <sys:String x:Key="ToolboxRepairWMI">Repair WMI component</sys:String>
-    <sys:String x:Key="ToolboxRepairWMIDescription">This will re-register the WMI component. Do not use it if you are unsure.&#x000A;&#x000A;After the repair is complete, you need to reboot your system for it to take effect.</sys:String>
+    <sys:String x:Key="ToolboxRepairVariablesDescription">Dadurch werden Ihre Systemvariablen auf die Standardwerte zurückgesetzt. Verwenden Sie sie nicht, wenn Sie sich nicht sicher sind.&#x000A;&#x000A;Nachdem die Reparatur abgeschlossen ist, müssen Sie Ihr System neu starten, damit sie wirksam wird.</sys:String>
+    <sys:String x:Key="ToolboxRepairWMI">WMI-Komponente reparieren</sys:String>
+    <sys:String x:Key="ToolboxRepairWMIDescription">Dadurch wird die WMI-Komponente neu registriert. Verwenden Sie sie nicht, wenn Sie sich nicht sicher sind.&#x000A;&#x000A;Nachdem die Reparatur abgeschlossen ist, müssen Sie Ihr System neu starten, damit sie wirksam wird.</sys:String>
```

Comment:
Verwenden Sie `dies` nicht, wenn Sie sich nicht sicher sind.
(wrong context, it's about the repair and not the component)

Reply:
ok, thanks